azext_iot/common/_azure.py

```diff
@@ -9,6 +9,25 @@
 from azext_iot.common.utility import validate_key_value_pairs
 from azext_iot.common.utility import trim_from_start
 from azext_iot._factory import iot_hub_service_factory
+from azure.cli.core._profile import Profile
+
+
+def get_aad_token(cmd, subscription=None, resource=None):
```

Comment:
This function is used by another function in the module `get_iot_central_tokens` and not from external (other modules in the package). It makes sense to hint as private i.e. prefix with `_`.